### PR TITLE
Manage a company

### PR DESCRIPTION
**Pull request by jerae-duffin**
New topic for companies. Will also include FAQs.

### CODE REVIEW

File: docker-hub/creating-companies.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+- The verified domains you want to move to the company level.
+- Confirm if you want to map one of your organization's SSO and SCIM settings to the company level.
+
+## Manage company owners
+
+As a company owner, you can configure Single Sign-on (SSO) and System for Cross-domain Identity Management (SCIM) for your organizations. This is only visible if your organization has a Docker Business subscription. If not, it's recommended that you first [upgrade your subscription](../subscription/upgrade.md).
+
+The SSO configuration updates all domain mappings for your organizations to a single domain so you can manage multiple organizations using one domain for your company. Group management is also available if your IdP supports group assignment.
+
+### Add  a company owner
```
Comment (craig-osterhout) [inline docker-hub/creating-companies.md:43]:
```suggestion
### Add a company owner
```

File: docker-hub/creating-companies.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+#### Okta mapping with SCIM enabled
+
+If users are signing in via SSO and SCIM, they’re also provisioned via SCIM.
+
+1. In Okta, select the **Provisioning** tab in the Docker application.
+2. In **Settings**, select **Integration**, and **Edit**.
+3. Enable **Push Groups**, and select **Save**.
+
+       > **Note**
```
Comment (craig-osterhout) [inline docker-hub/creating-companies.md:248]: This note and the following note appear as codeblocks in the preview. I'm not sure why though.

Comment (jerae-duffin) [reply docker-hub/creating-companies.md:248]: Thanks Craig! I'm updated it in my local will push the recent changes

File: docker-hub/creating-companies.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
+---
+description: manage companies
+keywords: company, multiple organizations, manage companies
+title: Manage a company
+---
+
+We're making it easier for large organizations with multiple businesses and groups to onboard their developers to use Docker products. If you have a Docker Business subscription and you want to setup your company, complete the following to set up your users, domains, and connect Docker to your IdP:
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:7]: Agree, updated! Thank you!

File: docker-hub/creating-companies.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+- Add a new company owner to the organization.
+- Add and verify your company domain.
+- Establish SSO connection between your identity provider (IdP) and Docker.
+- Sync your user data using System for Cross-Identity Management (SCIM)provisioning (group user management is now available).
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:12]: Updated

File: docker-hub/creating-companies.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+    >**Note**
+    >
+    > For example, the following company listed is **dockerinc** and the organization is **docker**.
+
+## Manage settings
+
+### Organizations with SSO connection
+
+1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"}, to view your company and organizations.
+2. On the **Organizations** page, you can create a new organization or select your company to access the **Overview** page.
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:72]: Thank you!  I agree it should say page. Once you select the company you land on the Organization page, although it has a tab that says Organization.

File: docker-hub/creating-companies.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+
+### Add a domain
+
+1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"} and navigate to the Organization tab and select your company.
+2. Select **Settings** to access the **Domain** and **SSO Connections**.
+3. Select **Add Domain** and continue with the on-screen instructions to add the **TXT Record Value** to your domain name system (DNS).
+
+    >**Note**
+    >
+    > It can take up to 48 hours to verify the domain. Select verify next to the domain in the domain table. Format your domains without protocol or www information, for example, yourcompany.com. This should include all email domains and subdomains users will use to access Docker. Public domains such as gmail.com, outlook.com, etc aren’t permitted. Also, the email domain should be set as the primary email.
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:99]: Updated

File: docker-hub/creating-companies.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+       > You can also enable Import Groups as an option if you want to import all Docker Hub teams into Okta groups.
+
+### Azure AD 
+
+#### Azure AD mapping with SCIM enabled
+
+1. Navigate to Azure AD and select your Docker application.
+2. Select the Provisioning tab and Edit provisioning.
+3. In Mappings, select Provision Azure Active Directory Groups, and select Yes to enable your mapping. 
+4. In the Target Object Actions, select Create, Update, and Delete.
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:259]: Thanks, this is updated

File: docker-hub/creating-companies.md [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+5. In the Attribute Mappings, confirm that the displayName, objectid, and members are all added.
```
[the remark this answers is not in the document]
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:260]: Updated here.

**Comment by jerae-duffin** [comment]
The FAQs at the bottom is a placeholder. I'll link it to the FAQ topic.

File: docker-hub/company-faqs.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+
+See Manage your [SCIM](../docker-hub/creating-companies.md#enable-scim) and [SSO](../docker-hub/creating-companies.md#single-sign-on-connection) settings. 
+
+### How does a company owner enable group mapping in my IdP?
+
+See [group mapping](../docker-hub/creating-companies.md#enable-sso-group-mapping) for your IdP.
+
+### What's the definition of a company vs an organization?
+
+A company is a collection of organizations that's together. An organization is a collection of repositories and teams that's managed together.
```
Comment (usha-mandya) [inline docker-hub/company-faqs.md:64]: > A company is a collection of organizations that's together

Is it missing something?

File: docker-hub/creating-companies.md [hunk 10]
```diff
@@ -0,0 +1,305 @@
...
+- Company owners don't take a seat.
+
+A company owner role is only available if your organization has a Docker Business subscription. If you don't have a Docker Business subscription, you must first [upgrade your subscription](../subscription/upgrade.md).
+
+## Get started
+
+You’ll need to send the following information to Docker to set up your company and send it to your CSM Docker team member:
+
+- The name of your company. For example, Docker uses the company name **dockerinc**.
+- The organizations want to associate with the new company.
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:37]: Should this be "The organizations that you want to associate with the new company."?

File: docker-hub/creating-companies.md [hunk 11]
```diff
@@ -0,0 +1,305 @@
...
+- The verified domains you want to move to the company level.
+- Confirm if you want to migrate one of your organization’s SSO and SCIM settings to the company. Migrating SSO settings will also migrate verified domains from the organization to the company parent.
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:39]: >...verified domains from the organization to the company parent.

Should this be parent company?

File: docker-hub/creating-companies.md [hunk 12]
```diff
@@ -0,0 +1,305 @@
...
+
+## Company overview and settings
+
+To navigate the company page:
+
+1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"}, to view your company and organizations.
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:45]:
```suggestion
1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"} to view your company and organizations.
```

File: docker-hub/creating-companies.md [hunk 13]
```diff
@@ -0,0 +1,305 @@
...
+2. On the **Organizations** page, you can create a new organization or select your company to access the **Overview** tab.
+
+    ![org-page](images/org-page.png){: width="700px" }
+
+## Manage company owners
+
+As a company owner, you can configure [Single Sign-on (SSO)](../single-sign-on/configure/index.md) and System for [Cross-domain Identity Management (SCIM)](../docker-hub/scim.md) for all organizations under the company. This is only visible if your organization has a Docker Business subscription. If you want to upgrade your subscription to include the organization under the company, see upgrade your subscription. If you want to upgrade your subscription to order to include your organization under your company, see [upgrade your subscription](../subscription/upgrade.md).
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:52]: There's a repetition at the end

File: docker-hub/creating-companies.md [hunk 14]
```diff
@@ -0,0 +1,305 @@
...
+
+The SSO configuration updates all domain mappings for your organizations to a single domain so you can manage multiple organizations using one domain for your company. Group management is also available if your IdP supports group assignment.
+
+### Add a company owner
+
+1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"}, navigate to the **Organizations** tab and select your company.
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:58]:
```suggestion
1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"}, navigate to the **Organizations** tab, and select your company.
```

File: docker-hub/creating-companies.md [hunk 15]
```diff
@@ -0,0 +1,305 @@
...
+
+1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"}, navigate to the **Organizations** tab, and select your company.
+2. In the **Overview** tab, find the company owner you want to remove.
+3. Select the **Action** icon and remove the owner.
+
+## Manage domains
+
+### Add a domain
+
+1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"} and navigate to the **Organization** page and select your company.
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:72]:
```suggestion
1. Sign in to [Docker Hub](https://hub.docker.com/){: target="_blank" rel="noopener" class="_"}, navigate to the **Organization** page, and select your company.
```

File: docker-hub/creating-companies.md [hunk 16]
```diff
@@ -0,0 +1,305 @@
...
+
+### Domains without SSO connection
+
+1. Add and [verify your domain](../single-sign-on/configure/index.md#domain-verification).
+2. Create an SSO connection between Docker and your IdP. You aren't required to enforce SSO login for your users.
+3. Review the [requirements](../single-sign-on/requirements/index.md) before configuring SSO.
+
+## FAQs
+
+To learn more see [FAQs](../docker-hub/company-faqs.md).
```
Comment (usha-mandya) [inline docker-hub/creating-companies.md:305]:
```suggestion
For frequently asked questions, see [FAQs](../docker-hub/company-faqs.md).
```

Comment (jerae-duffin) [reply docker-hub/creating-companies.md:305]: TY! Updated

File: docker-hub/creating-companies.md (hunk 13 above)
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:52]: Removed the duplicate

File: docker-hub/creating-companies.md (hunk 12 above)
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:45]: Updated

File: docker-hub/creating-companies.md (hunk 11 above)
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:39]: Yes, thank you!

File: docker-hub/creating-companies.md (hunk 10 above)
Comment (jerae-duffin) [reply docker-hub/creating-companies.md:37]: Updated

File: docker-hub/company-faqs.md (hunk 9 above)
Comment (jerae-duffin) [reply docker-hub/company-faqs.md:64]: Updated

**Comment by jerae-duffin** [comment]
WIP/ adding new images

File: docker-hub/enforcing-sso.md [hunk 17]
```diff
@@ -0,0 +1,18 @@
...
+Without SSO enforcement, users can continue to sign in using Docker username and password. If users login with your Domain email, they will authenticate through your identity provider instead. 
+
+You must test your SSO connection first if you’d like to enforce SSO log-in. All users must authenticate with an email address instead of their Docker ID if SSO is enforced
+
+
+1. In the **Single Sign-On Connections** table, select the Action icon and **Enforce Single Sign-on**.
+
+    > **Note**
+    >
+    > When you enforce SSO,  all members of your organization with a matching domain must authenticate through your IdP.
```
Comment (craig-osterhout) [inline docker-hub/enforcing-sso.md:16]:
```suggestion
    > When you enforce SSO, all members of your organization with a matching domain must authenticate through your IdP. 
```

File: docker-hub/company-faqs.md [hunk 18]
```diff
@@ -0,0 +1,70 @@
...
+Domain verification, Single Sign-on, and System for Cross-domain Identity Management (SCIM) are supported at the company level. The following aren't supported:
+
+- Image Access Management
+- Registry Access Management
+- User management
+- Billing
+
+### What's required to create a company name?
+
+A company name must be unique to that of it's child organization. If a child organization requires the same name as a company, we suggest modifying slightly. For example, **Docker Inc** (parent company), **Docker** (child organization).
```
Comment (craig-osterhout) [inline docker-hub/company-faqs.md:46]:
```suggestion
A company name must be unique to that of its child organization. If a child organization requires the same name as a company, we suggest modifying slightly. For example, **Docker Inc** (parent company), **Docker** (child organization).
```

File: docker-hub/enforcing-sso.md (hunk 17 above)
Comment (jerae-duffin) [reply docker-hub/enforcing-sso.md]: Updated in new commit

File: docker-hub/company-faqs.md (hunk 18 above)
Comment (jerae-duffin) [reply docker-hub/company-faqs.md]: Updated in new commit

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**